core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -65,7 +65,12 @@ default boolean schemaExists(ConnectorSession session, String schemaName)
     }
 
     /**
-     * Returns a table handle for the specified table name, or null if the connector does not contain the table.
+     * Returns a table handle for the specified table name, or {@code null} if the connector does not contain the table.
+     * Implementation should return {@code null} when {@code tableName} refers to other relation type (e.g. a view, or
+     * materialized view).
```

Comment:
We could mention what should be implement for view or materialized view.

Reply:
added `@see`